sentinel/package.json

```diff
@@ -27,7 +27,7 @@
     "request-promise-native": "^1.0.9",
     "url-join": "^4.0.1",
     "uuid": "^8.3.2",
-    "winston": "^3.3.3"
+    "winston": "^3.6.0"
```

Comment:
Thanks for noticing this! Very helpful.

So, the [winston issue](https://github.com/winstonjs/winston/issues/1345) is:
> Using a custom format that uppercases the level with colorize will cause the log line output to have lots of empty space before it (clears my console so probably 50+ empty lines).

which in turn cites:

> this should be fixed [...so you...] can solve this by:
> ```javascript
> const custom = format.printf((info) => {
>   return `${info.level}: ${info.message}`;
> });
> 
> const log = createLogger({
>   format: format.combine(
>     format(info => {
>       info.level = info.level.toUpperCase()
>       return info;
>     })(),
>     format.colorize(),
>     custom
>   ),
>   transports: [new transports.Console()]
> });

And I honestly don't know what we're supposed to change in [our code](https://github.com/medic/cht-core/blob/c51127ba75ae89e7d6f8d0f5ea676f09a226dddd/sentinel/src/lib/logger.js#L64):

```javascript
format: format.combine(
    // https://github.com/winstonjs/winston/issues/1345
    format(info => {
      info.level = info.level.toUpperCase();
      return info;
    })(),
    format.colorize(),
    format.timestamp({
      format: 'YYYY-MM-DD HH:mm:ss',
    }),
    format.printf(info => `${info.timestamp} ${info.level}: ${info.message} ${info.stack ? info.stack : ''}`)
),
```

Otherwise, I'll try and dig into how to test [we're safely sanitizing the log](https://github.com/medic/cht-core/blob/c51127ba75ae89e7d6f8d0f5ea676f09a226dddd/sentinel/src/lib/logger.js#L24) per your other point.  Thanks again!

Reply:
I had a look to their PR and we're probably fine. I didn't spot anything wrong in the sentinel logs by regular run